### PR TITLE
ADA-1800: expand energy plans controller

### PR DESCRIPTION
**Pull request by leo-adatree**

### CODE REVIEW

File: data.yaml [hunk 1]
```diff
@@ -6132,7 +6206,6 @@ components:
       description: User Agent header of the consumer facing application. Mandatory for consumer present calls using a Machine token.
       schema:
         type: string
-        example: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.0.0 Safari/537.36
```
Comment (arthuradr) [inline data.yaml]: is this removal intended?

Comment (leo-adatree) [reply data.yaml]: it's an exactly replica of our internal doc

Comment (arthuradr) [reply data.yaml]: I see, this parameter is shared by different services and Banking does not have the example, can we please keep it here and sync this one back to banking?

Comment (leo-adatree) [reply data.yaml]: Done
